setup.py

```diff
@@ -10,25 +10,29 @@
     install_requires=[],
     entry_points={
         'console_scripts': [
-            # Boundaries
             'extract_adm0_cod=extract.cod:get_adm0_snakemake',
             'extract_adm1_cod=extract.cod:get_adm1_snakemake',
             'extract_adm2_cod=extract.cod:get_adm2_snakemake',
             'extract_adm3_cod=extract.cod:get_adm3_snakemake',
+            'extract_roads_cod=extract.cod:get_roads_snakemake',
```

Comment:
The roads COD entrypoints are already defined on lines 36 and 36, the duplicates need to be removed or else `pip install .` fails